Bibliography.bib

```diff
@@ -0,0 +1,2 @@
+UN - Understanding Machine Learning:From Theory to Algorithms, 2014 by Shai Shalev-Shwartz and Shai Ben-David,
```

Comment:
add a ROOT file and a document/root.bib file instead of this, 
then you can use @{cite UnderstandingML} in your theories